src/js/components/Stack/Stack.js

```diff
@@ -17,14 +17,18 @@ class Stack extends Component {
     } else if (guidingIndex === 'last') {
       guidingIndex = React.Children.count(children) - 1;
     }
-    const styledChildren = Children.map(children, (child, index) => {
+    let childIndex = 0;
+    const styledChildren = Children.map(children, (child) => {
       if (child) {
-        if (index === guidingIndex) {
-          return <StyledStackLayer guiding={true}>{child}</StyledStackLayer>;
+        let layer = <StyledStackLayer anchor={anchor} fill={fill}>{child}</StyledStackLayer>;
```

Comment:
I don't think we want `fill` on the non-guiding layer. They should be driven just by the anchor, I think?